spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -379,7 +379,7 @@ def certificate_valid?
 
       # @return (Bool) Is the current provisioning profile valid?
       def valid?
```

Comment:
from doing a find all on the project, this method is only called in runner.sh and the specs.  So it shouldn't effect any other systems.